master/getting-started/bare-metal/installation/binary-mgr.md

```diff
@@ -3,14 +3,6 @@ title: Binary install with package manager
 canonical_url: 'https://docs.projectcalico.org/v3.3/getting-started/bare-metal/installation/binary-mgr'
 ---
 
-
-## Prerequisites
```

Comment:
They do need calicoctl to complete this procedure, no?

Reply:
Yes - and I believe I've covered that now in the index.md page.  (Because in fact calicoctl is needed for all the methods, so it makes sense to say that upfront.)

Reply:
Oh oops, missed that. OK Awesome